Test/VimCoreTest/NormalModeTest.cs

```diff
@@ -183,6 +183,18 @@ public void CanProcess_PrintableNonAscii()
             Assert.False(_mode.CanProcess(KeyInputUtil.CharToKeyInput('´')));
         }
 
+        /// <summary>
+        /// Verify that normal mode can process both C-LeftMouse and C-LeftRelease
+        /// </summary>
+        [WpfFact]
+        public void CanProcess_ControlClickAndRelease()

```

Comment:
Consider adding a test that verifies this action doesn't impact repeat last command (the `.` operator).

Reply:
Good idea!  Added one.